src/data/hcp.json

```diff
@@ -5,7 +5,7 @@
 		"indexName": "product_HCP",
 		"searchOnlyApiKey": "1162c7d22059acafddb62badfdf68f35"
 	},
-	"basePaths": ["docs"],
+	"basePaths": ["docs", "api-docs/packer"],
```

Comment:
❓ why not just use `/api-docs`?

Reply:
It's because there aren't any other pages under `/hcp/api-docs` that are valid, only `/hcp/api-docs/packer`.